db/app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+
+const { getTopics } = require('./controllers/topics.controller.js')
+
+app.use(express.json())
```

Comment:
Please remove this express.json(). We shouldn't have this here until it's needed 💭